README.md

```diff
@@ -1,137 +1,104 @@
 
 # JitBench
 
+![Build Status](https://travis-ci.org/aspnet/JitBench.svg?branch=rel/2.0.0)             
+
 A repository for aspnet workloads suitable for testing the JIT.
 
 Right now there is only one workload app here, at `src/MusicStore`
 
-## Branches
```

Comment:
@noahfalk - IIRC from our conversation you intend on always using the last stable release of ASP.NET Core - but with the newest shared framework/runtime and SDK correct?

So this panoply is totally uninteresting right?

Reply:
Yes, though I didn't want to be too aggressive about culling it before people have had a chance to verify the new workflow works for them. 

The script parameterizes the version of ASP.NET, so in theory regardless of which version we target I hope that it can be done with only a single branch. In practice I expect most of our testing will be done against last stable ASP.Net release, or occasionally a newer dev version if we believe it will have significantly different perf characteristics.

Reply:
I did a purge of some legacy stuff last week. 

The only branches I'm aware of anyone using are dev, rel/2.0.0 and master.

re: ASP.NET versioning - I just want to make sure we don't lag too far behind publicly released bits. Usually an update will include some code changes as well as just bumping up the libraries.

I don't have a concern about updating after things ship. We can iron out the logistics of doing that once 2.1 ships.

As an aside, we do expect 2.1.0 to have different perf characteristics 😆